modules/programs/git.nix

```diff
@@ -133,6 +133,10 @@ in
         '';
         description = "List of configuration files to include.";
       };
+
+      lfs = {
+        enable = mkEnableOption "Git Large File Storage";
```

Comment:
We should probably add an option `skipSmudge` which does the following

>  Skips automatic downloading of objects on clone or pull. This requires a
    manual "git lfs pull" every time a new commit is checked out on your
    repository.

I think lots of people, myself included, use this global configuration.

It just adds a `--skip` to the following:
```
smudge = git-lfs smudge --skip -- %f
process = git-lfs filter-process --skip
```

Reply:
btw, you'd get this config if you issued `git lfs install --skip-smudge`.

Reply:
A good idea, I've added a `skipSmudge` option.